changes to episode 6, tasks 11-23 (#148)
carpentries-incubator/high-dimensional-stats-r: _episodes_rmd/06-k-means.Rmd

>
{: .callout}

> # K-means clustering applied to the single-cell RNA sequence data

Reviewer: ```suggestion
# K-means clustering applied to the single-cell RNA sequencing data
```

Just semantics, but I might say "sequence data" if we were interested in the actual sequences/reads, rather than counts. For things with just read counts, I'd say sequencing data

Let's carry out K-means clustering in `R` using some real high-dimensional data.

> We're going to work with single-cell RNA sequence data, `scranseq`, in these clustering challenges,

Reviewer: ```suggestion
We're going to work with single-cell RNA sequencing data, "scRNAseq", in these clustering challenges,
```

allows us to visualise this incredibly complex data in a small number of
dimensions. In this case, we'll be using principal component analysis (PCA) to
compress the data by identifying the major axes of variation in the data,
before running our clustering algorithms on this lower-dimensional data to explore
the similarity of features.

The `scater` package has some easy-to-use tools to calculate a PCA for
`SummarizedExperiment` objects.

> Let's load the `scrnaseq` data and calculate some principal components.

Reviewer: ```suggestion
Let's load the scRNAseq data and calculate some principal components.
```

the underlying data, representing the two largest axes of variation. 

We can now run K-means clustering on the first and second principal components

> of the `scrnaseq` data using the `kmeans` function.

Reviewer: ```suggestion
of the scRNAseq data using the `kmeans` function.
```
